### PR TITLE
Braces

### PR DESCRIPTION
**Pull request by nemchik**
Fixes #

## Proposed Changes

- Format markdown files (mostly README.md files in all subfolders)
- Adjust yaml formatting (github actions files)
- Use `${VAR}` instead of `$VAR` in shell scripts and others known to support this syntax.

## Readiness Checklist
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by admiralAwkbar** [comment]
@nemchik your doing the dirt work... I appreciate it...

**Comment by nemchik** [comment]
Ok, most of this was regex replace and tooling, but i have manually reviewed all of the changes and double checked them. Should be good to go.

**Comment by nemchik** [comment]
Added the only tag that seemed to fit lol

File: lib/buildFileList.sh [hunk 1]
```diff
@@ -86,26 +86,26 @@ function BuildFileList() {
     ###########################
     # Extract just the file and extension, reverse it, cut off extension,
     # reverse it back, substitute to lowercase
-    FILE_TYPE=$(basename "$FILE" | rev | cut -f1 -d'.' | rev | awk '{print tolower($0)}')
+    FILE_TYPE=$(basename "${FILE}" | rev | cut -f1 -d'.' | rev | awk '{print tolower(${0})}')
```
Comment (GaboFDC) [inline lib/buildFileList.sh:89]: I think this broke the command, can you check?
This also shows that we don't have test for when `VALIDATE_ALL_CODEBASE` is false.
Kind of poetic that @admiralAwkbar broke the awk 😂

**Pull request merged**